### PR TITLE
README French translation

### LINKED ISSUE #1337: [Translate] README Spanish translation
### Summary

As suggested by @malvikasharan in #1335, @camachoreina and I we will be translating the README to Spanish.
### What needs to be done?

- [ ] *README.md translation*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### LINKED ISSUE #1339: [Translate] README French translation
### Summary

As requested in #1335, I will be translating the README to French.

### What needs to be done?

 README.md translation

### PR DESCRIPTION
**Pull request by jessyprovencher**
### Summary

Fixes #1339

### List of changes proposed in this PR (pull-request)

* *Add French translation for README.md (see file README-French.md)*

### What should a reviewer concentrate their feedback on?

Reviewer should be French-speaker.

- [ ] Typos / Phrasing
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by SamGuay** [comment]
:wave:  @jessyprovencher 

Are you still looking for a french-speaking reviewer ? 

Let me know if you want me to review it. 

Thanks

**Comment by jessyprovencher** [comment]
@SamGuay 

Yes, i'm still looking for a french-speaking reviewer. You can review it anytime! 🎉

**Comment by malvikasharan** [comment]
@SamGuay I am marking this PR for hacktoberfest accepted to make sure @jessyprovencher gets to count this towards his 4 PR :), while I let you review it in your time.

**Comment by KirstieJane** [comment]
Hi @jessyprovencher ! @SamGuay and I are just having a look at this PR now - I wonder if you could make sure that you're up to date with master? It feels like there are a lot of commits in this branch that shouldn't be showing up 😕 I think maybe its GitHub being a little strange. Ping me if you aren't sure how to do that though!

File: README-French.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+[![](https://img.shields.io/static/v1?label=TuringWay&message=Je%20veux%20contribuer!&color=yellow&logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8%2F9hAAACYklEQVQ4jXXTy09TQRTH8f5VPhI1xoVxYURNAFcmRleaGDdGXQlKAYkLUARNfICoScGKpTyE3t5bkKD2AUQepUXB0gcgLTalD9rema8LKRVrT3I2k%2Fl95kwyY6BMfQiFqHaoVDlUBoJBZJl9hn8XRsIhqh0abd55tnWdrBA8WfBSpakMhUqhXUCJhKl2aLR65%2FEtLeGc%2BYoy5aHf46bX7cThctK%2BAw2HQkVAW41wzqHRMjNNRteR%2BQzGjg5udZtQ47FiO50gdLZ1nVbvPNUOFSUSxnB4sJ%2F0TjCTTjHk%2BoJl%2BRtqPEaL6zMH79Rw0dyDVVURqRgyn0EkN8jkshwZGsBQodgQyQ2kyDPsce859drjdqLRKE0D%2FZhHR5F6DpHc2B3%2FjF3BcFqxARIpBXXmt9ii67vAYDhIr8fNx0UfE3OzzC0sIHIpxNYqSPEHqFBsiFQMkU3h8vs5%2FvABTeNje6BCj%2FxcwzLlIZHYROq5v4EoIr2JyCbJ57Kobjd3u7o41v4I68pyCfTGrhSvUKHYAJD5bcTWGjKbJJdO4A8E6JyexP4rWgK8Vkb2AjK7hcxnmZybxfF9kff%2BhZJQofvXwhg7O4vAfU2l79ME79xOrjY3c9ZYVzZs8nvZf6%2BRQCRCTgiODg1iCK6vc6WtjZM1tzlRW8sNa99%2Fx64fH%2BNAQz0un49nfh%2BVmspAcKX4lKWUbMbjXOg2cf3Vy%2BLIoRWqekxc7nhB6%2FQ0lZqKJRBAyjKfKZFIcKixgVPPn3LTamFfUyPne7qp1Oz0Bn4g5d7vVAIUamJ2FqPZzCW7gvlHabBQvwE2XnlAiFRrOwAAAABJRU5ErkJggg%3D%3D)](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)
+
```
Comment (SamGuay) [inline README-French.md:8]:
```suggestion
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![Tous les contributeurs et contributrices](https://img.shields.io/badge/tous_les_contributeurs_et_contributrices-206-orange.svg)](#contributors)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

**Comment by jessyprovencher** [comment]
> 
> 
> Hi @jessyprovencher ! @SamGuay and I are just having a look at this PR now - I wonder if you could make sure that you're up to date with master? It feels like there are a lot of commits in this branch that shouldn't be showing up 😕 I think maybe its GitHub being a little strange. Ping me if you aren't sure how to do that though!

I tried a few weeks ago to update with master branch but look like i've made a mistake... (not quite skilled with large Git repos) I'd like some help to clear those up and update with the changes made. Hope to hear from you soon! 🎉

**Comment by SamGuay** [comment]
I'll leave the cleaning part to @KirstieJane. 

As for the review, I'll probably be able to finish it by tomorrow or by wednesday. I'm working on it for the The Turing Way | Book Dash November 2020 before diving into finding the best inclusive writing guide! I'll make a couple of suggestions and if my suggestions are not clear enough or you think we should improve more, we can always have a chat. Overall, it seems pretty good, so there should be much to do! 

Thanks again for your contribution :muscle:

**Review by SamGuay — CHANGES_REQUESTED** [review]
Here are my suggestions. Feel free to adapt them as you see fit.

File: README-French.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![Tous les contributeurs et contributrices](https://img.shields.io/badge/tous_les_contributeurs_et_contributrices-206-orange.svg)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+**_Ce fichier README.md est également disponible en anglais ([README-English](https://github.com/alan-turing-institute/the-turing-way), espagnol ([README-Spanish](/README-translated/README-Spanish.md)), italien ([README-Italian](/README-translated/README-Italian.md)), néerlandais ([README-Dutch](/README-translated/README-Dutch.md)) et portuguais ([README-Portuguese](/README-translated/README-Portuguese.md)) (ordonnés alphabétiquement)._**
+
+_The Turing Way_ est un guide légèrement avisé de la science des données reproductibles.
+Vous pouvez le lire ici: <https://the-turing-way.netlify.com>
+Vous consultez actuellement le dépôt GitHub du projet où tous les éléments qui composent le guide sont en ligne et où se déroule le processus d'écriture / de construction du guide.
```
[the remark this answers is not in the document]
Comment (jessyprovencher) [reply README-French.md:17]:
```suggestion
Vous consultez actuellement le dépôt GitHub du projet dans lequel se trouve tous les éléments qui composent le guide et ses processus d'écriture / de construction.
```

**Review by SamGuay — APPROVED** [review]
All is good now.

**Comment by KirstieJane** [comment]
Hi folks!! I've accidentally done something super wild and accidentally lost @jessyprovencher's changes in HIS fork of the repository. BAD KIRSTIE.

However, I had made a different error earlier and had pushed the changes to a branch in this repository. That PR is open now at #1590.

My suggestion is that we keep this PR closed, and we merge #1590 if all the changes are ready to go.

Many apologies for my wild git-ing 🙈  and thank you SO MUCH for this fantastic work!

**Pull request closed without merging**